libs/src/sdk/oauth/OAuth.ts

```diff
@@ -171,7 +215,15 @@ export class OAuth {
     }, 500)
   }
 
-  renderButton(element: HTMLElement, options?: ButtonOptions) {
+  renderButton({
+    element,
+    scope = 'read',
+    buttonOptions
+  }: {
+    element: HTMLElement
+    scope: OAuthScope
+    buttonOptions?: ButtonOptions
```

Comment:
I think this might be the main breaking change? if so, we can probably avoid the breaking change by leaving this named `options` if we wanted

Reply:
before, it was two separate params not one object param unfort :( 
will look into function overloading to prevent breaking change

Reply:
Ah I see, nevermind then. No harm in breaking changes! Just bump the package.json so we don't forget